src/handlers.js

```diff
@@ -417,72 +585,82 @@ async function jobHandler(message) {
     }
   }
 
-  try {
-    taskGroupId = graphConfig.tasks[0].task.taskGroupId;
-    debug(`Creating tasks for ${organization}/${repository}@${sha} (taskGroupId: ${taskGroupId})`);
-    await this.createTasks({scopes: graphConfig.scopes, tasks: graphConfig.tasks});
+  taskGroupId = graphConfig.tasks[0].task.taskGroupId;
```

Comment:
taskIds?

Reply:
ah, I see. Yes, good point

Reply:
I am leaving this as it is for now. I would like to tackle this in a separate follow-up PR, because this is something that I should have done when I added v1.